pandas/io/gbq.py

```diff
@@ -27,6 +27,7 @@ def read_gbq(
     use_bqstorage_api=None,
     private_key=None,
     verbose=None,
+    progress_bar_type="tqdm",
```

Comment:
Let's default to `None` until there have been a few more `pandas-gbq` releases with this feature.

```suggestion
    progress_bar_type=None,
```

Reply:
changed!! thanks!!